JeMPI_Apps/JeMPI_Linker/src/main/java/org/jembi/jempi/linker/BackEnd.java

```diff
@@ -100,6 +109,19 @@ static void updateGoldenRecordField(
             final var result = libMPI.updateGoldenRecordField(goldenId, fieldName, maxEntry.getKey());
             if (!result) {
                LOGGER.error("libMPI.updateGoldenRecordField({}, {}, {})", goldenId, fieldName, maxEntry.getKey());
+            }  else {
+               // Update scores of all associated entities
+               mpiPatientList.forEach(mpiPatient -> {
```

Comment:
@mahaomolise92 I think this can be refactored and placed in its own private reCompute method in Backend